hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -312,8 +312,8 @@ public Authorizer authorizer() {
     }
 
     @Override
-    public Instant currentTime() {
-        return consensusNow();
+    public int numSigs() {
```

Comment:
Sure, hopefully that'll also help to prevent it from being misused (e.g. in query situations). This is what it was called in mono and I blindly ported it.

Reply:
Done